packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -87,6 +88,69 @@ const ShareActivity = WebexPlugin.extend({
     }
   },
 
+  /**
+   * Adds an additional GIF to the share activity
+   * Different from regular add to skip uploading to webex files service
+   * @param {File} gif
+   * @param {Object} options
+   * @param {Object} options.actions
+   * @returns {Promise}
+   */
+  addGif(gif, options) {
+    let gifToAdd = this.uploads.get(gif);
+
+    // If the gif already exists, then don't do anything
+    if (gifToAdd) {
+      return Promise.resolve();
+    }
+
+    gifToAdd = Object.assign({
+      displayName: gif.name,
+      fileSize: gif.size || gif.byteLength || gif.length,
+      mimeType: gif.type,
+      url: 'https://giphy.com',
+      objectType: 'file',
+      [FILE_SYMBOL]: gif
+    }, pick(options, 'actions'));
+
+    this.uploads.set(gif, gifToAdd);
+
+    return SCR.create().then((scr) => {
```

Comment:
Can you also write a comment explaining what this function and SCR is doing.
I have no clue what SCR is and why we're it's necessary to install a new package